control-plane/grpc/proto/v1/ha/node_agent.proto

```diff
@@ -21,4 +22,26 @@ message ReplacePathRequest {
 
   // Pubish context of the volume
   common.MapWrapper publish_context = 3;
+}
+
+// Get Controller request message.
+message GetNvmeControllerRequest {
+  // Nvme path containing nqn for the volume.
+  string nvme_path = 1;
+}
+
+// Get Controller response message.
+message GetNvmeControllerResponse {
+  oneof reply {
+    // List of target address registered as Nvme Subsystem.
+    NvmeControllers nvme_controllers = 1;
+    // Error incase No subsystem found.
+    common.ReplyError error = 2;
```

Comment:
q: do we need a ReplyError error here, is the normal grpc status not enough?

Reply:
We are sending ReplyError::not_found from node agent incase we didnt find any subsystem for nqn or any other error. In cluster agent, We match if its ReplyError::not_found . If yes, We cleanup all shutdown targets for the volume.